python/create_mask.py

```diff
@@ -0,0 +1,138 @@
+"""
+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
+;
+; Name        : create_mask
+;
+; Purpose     : Create transparent CH masks from previously stored data
+;
+; Syntax      : create_mask.py path 
+;
+; Inputs      : CHIMERA ch_location.txt file
+;
+; Outputs     : Transparent CH mask
+;
+; Keywords    : path = directory path of file	
+;
+; Example     : ipython> % run -i create_mask.py '/path/to/file/location/'
+;
+; Required Packages    : pip install mahotas
+;
+; History     : v1.0, Written 14-Jun-2018, Tadhg Garton, TCD
+;
+; Contact     : gartont@tcd.ie
+;		info@solarmonitor.org
+
+"""
+
+from datetime import datetime
+import glob
+import sys
+
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+import os.path as op
```

Comment:
This should be in the above block as it's a core module.
Also, `op` is a non standard way of abbreviating this. I wouldn't do so.